app/models/shipit/pull_request.rb

```diff
@@ -111,11 +112,11 @@ def self.schedule_merges
     def self.extract_number(stack, number_or_url)
       case number_or_url
       when /\A#?(\d+)\z/
-        $1.to_i
+        Regexp.last_match(1).to_i
```

Comment:
That cop is total 🐄 💩 . It's just as global as before, just way more verbose.

I know it's in the style guide, but let's disable it.